gamestonk_terminal/jupyter/reports/reports_controller.py

```diff
@@ -133,7 +133,7 @@ def switch(self, an_input: str):
 
         (known_args, other_args) = self.parser.parse_known_args(an_input.split())
 
-        if not other_args and an_input in ("2", "3", "5"):
+        if not other_args and an_input in ("1", "3", "4"):
```

Comment:
@piiq What were you trying to achieve this change. It actually broke the older reports :( . If I changed it back, both the new & old reports work.

Reply:
ah yes, pls do